pom.xml

```diff
@@ -111,6 +111,11 @@
       <artifactId>slf4j-api</artifactId>
       <version>1.8.0-beta2</version>
     </dependency>
+    <dependency>
+      <groupId>com.jayway.jsonpath</groupId>
+      <artifactId>json-path</artifactId>
+      <version>2.4.0</version>
```

Comment:
Any way to do this without an extra dependency?
Jackson already uses `JsonPointer`s, maybe that could be used to specify ignored elements as well?